docs/examples.rst

```diff
@@ -53,29 +48,32 @@ Example: synthesizing 1000 soundscapes in one go
     time_stretch_min = 0.8
     time_stretch_max = 1.2
 
+    # generate a random seed for this Scaper object
+    seed = np.random.randint(0, 100000)
+
+    ## alternate ways to define random state:
+    #     seed = np.random.RandomState(0)
+    #
+    # or don't define any random state. runs will be random and not reproducible in 
+    # this case. you can use np.random.get_state() to reproduce the run after the fact
+    # if needed:
+    #     seed = None
+
+    # create a scaper that will be used below
+    sc = scaper.Scaper(duration, fg_folder, bg_folder, random_state=seed)
+    sc.protected_labels = []
+    sc.ref_db = ref_db
+
     # Generate 1000 soundscapes using a truncated normal distribution of start times
 
     for n in range(n_soundscapes):
 
         print('Generating soundscape: {:d}/{:d}'.format(n+1, n_soundscapes))
-        # generate a random seed for this Scaper object
-        # because of the earlier seed these will always be the same and are given to the
-        # Scaper object for reproducible results.
-
-        seed = np.random.randint(0, 100000)
-
-        ## alternate ways to define random state:
-        #     seed = np.random.RandomState(0)
-        #
-        # or don't define any random state. runs will be random and not reproducible in 
-        # this case. you can use np.random.get_state() to reproduce the run after the fact
-        # if needed:
-        #     seed = None
-
-        # create a scaper
-        sc = scaper.Scaper(duration, fg_folder, bg_folder, random_state=seed)
-        sc.protected_labels = []
-        sc.ref_db = ref_db
+
+        # reset the event specifications for foreground and background at the beginning
+        # of each loop.
```

Comment:
reset the event specifications for foreground and background at the beginning of each loop to clear all previously added events